pyspeckit/spectrum/showspec.py

```diff
@@ -585,22 +587,22 @@ def __call__(self, interactive=False, usemoments=True, fitcolor='r',
             self.guesses = []
             self.click = self.specplotter.axis.figure.canvas.mpl_connect('button_press_event',self.makeguess)
             self.autoannotate = annotate
-        elif multifit:
+        else:
             if guesses is None:
                 print "You must input guesses when using multifit.  Also, baseline first!"
             else:
                 self.guesses = guesses
                 self.multifit()
                 self.autoannotate = annotate
-        else:
-            #print "Non-interactive, 1D fit with automatic guessing"
-            if self.specplotter.baseline.order is None:
-                self.specplotter.baseline.order=0
-                self.onedfit(usemoments=usemoments,annotate=annotate,**kwargs)
-            else:
-                self.onedfit(usemoments=usemoments,annotate=annotate,
-                        vheight=False,height=0.0,**kwargs)
-            if self.specplotter.autorefresh: self.specplotter.refresh()
+        # else:
+        #     #print "Non-interactive, 1D fit with automatic guessing"
+        #     if self.specplotter.baseline.order is None:
+        #         self.specplotter.baseline.order=0
+        #         self.onedfit(usemoments=usemoments,annotate=annotate,**kwargs)
+        #     else:
+        #         self.onedfit(usemoments=usemoments,annotate=annotate,
+        #                 vheight=False,height=0.0,**kwargs)
+        #     if self.specplotter.autorefresh: self.specplotter.refresh()
```

Comment:
`showspec` was an even earlier iteration of pyspeckit; it is OK to comment out lines like this (I am just confirming that what you've done here is OK).